### PR TITLE
Fixed --echo option is not used by clickhouse-client in batch mode with single query

### PR DESCRIPTION
**Pull request by nkolotov**
Fix for https://github.com/ClickHouse/ClickHouse/issues/32842

Changelog category (leave one):
- Bug Fix (user-visible misbehaviour in official stable or prestable release)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fixed --echo option is not used by clickhouse-client in batch mode with single query

### CODE REVIEW

**Comment by evillique** [comment]
@Mergifyio update

**Comment by alexey-milovidov** [comment]
> curl: (6) Could not resolve host: s3.amazonaws.com

Most likely the AWS region is down.

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Maybe add a test?

File: tests/queries/0_stateless/02151_client_option_echo.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# single query echo on
+${CLICKHOUSE_CLIENT} --echo --query="DROP TABLE IF EXISTS echo_test_0" && \
+# single query echo off
+${CLICKHOUSE_CLIENT} --query="DROP TABLE IF EXISTS echo_test_1" && \
+# multi query echo on
+${CLICKHOUSE_CLIENT} --echo --multiquery --query="DROP TABLE IF EXISTS echo_test_2;DROP TABLE IF EXISTS echo_test_3" && \
+# multi query echo off
+${CLICKHOUSE_CLIENT} --multiquery --query="DROP TABLE IF EXISTS echo_test_4;DROP TABLE IF EXISTS echo_test_5" && \
+
+true
```
Comment (evillique) [inline tests/queries/0_stateless/02151_client_option_echo.sh:12]: What does this line do?

Comment (nkolotov) [reply tests/queries/0_stateless/02151_client_option_echo.sh:12]: `true` is supposed to be ending the chain of commands connected with `&&`. I think it is a common practice when writing chained commands so all of them will end with `&& \` and can be easily rearranged. The stray blanc line probably got there by accident. Is command chaining actually required for these tests? 

Also i see that Fast test (actions) failed with message `line 4: --echo: command not found`. Perhaps `CLICKHOUSE_CLIENT` did not expand properly? Or perhaps this test must be registered somewhere to use correct environment?

File: tests/queries/0_stateless/02151_client_option_echo.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```
Comment (evillique) [inline tests/queries/0_stateless/02151_client_option_echo.sh:2]:
```suggestion

CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
# shellcheck source=../shell_config.sh
. "$CURDIR"/../shell_config.sh

```

File: tests/queries/0_stateless/02151_client_option_echo.sh (hunk 1 above)
Comment (evillique) [reply tests/queries/0_stateless/02151_client_option_echo.sh:12]: Oh, I missed that commands are chained. So no, there is no need to chain them, the errors will be written to stderr and if something goes wrong the test will fail anyway :)

I suggested a change and your test should pass with it

Comment (evillique) [reply tests/queries/0_stateless/02151_client_option_echo.sh:12]: Oh, I missed that commands are chained. So no, there is no need to chain them, the errors will be written to stderr and if something goes wrong the test will fail anyway :)

I suggested a change and your test should pass with it

Comment (nkolotov) [reply tests/queries/0_stateless/02151_client_option_echo.sh]: Alright, i've applied these changes and removed chaining.

**Pull request merged**